src/app/pages/FastBtcPage/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect } from 'react';
+import { useHistory, useParams } from 'react-router';
+import { Helmet } from 'react-helmet-async';
+import { useTranslation } from 'react-i18next';
+
+import { translations } from 'locales/i18n';
+import { useAccount } from '../../hooks/useAccount';
+import { WithdrawContainer } from './containers/WithdrawContainer';
+import { DepositContainer } from './containers/DepositContainer';
+import Header from './components/Header';
+import classNames from 'classnames';
+import styles from './fast-btc-page.module.css';
+
+type FastBtcDirectionType = 'deposit' | 'withdraw';
```

Comment:
Make it an `enum` pls because you need to hardcode 'deposit' and 'withdraw' in several places.